docs/guides/_toc.json

```diff
@@ -637,6 +637,10 @@
             {
               "title": "Visualize results",
               "url": "/docs/guides/visualize-results"
+            },
+            {
+              "title": "Explore gates and circuits with IBM Quantum Composer",
```

Comment:
Do we want the left ToC to be such  long title?

Reply:
I've pinged Abby to weigh in